dynet/expr.h

```diff
@@ -1090,6 +1090,18 @@ Expression nobackprop(const Expression& x);
 
 /**
  * \ingroup flowoperations
+ * \brief Negative backprop 
+ * \details This node has no effect on the forward pass, but takes negative on backprop process. 
+ *          This is useful when you want to create a domain adaption network
```

Comment:
I think "domain adaptation network" may be hard to understand here. What about saying something like "This operation is widely used in adversarial networks."